app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,10 +1,44 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url("https://use.typekit.net/hfn2egb.css");
 
 // Define fonts for body and headers
-$body-font: "Work Sans", "Helvetica", "sans-serif";
-$headers-font: "Nunito", "Helvetica", "sans-serif";
+$body-font: "area-normal", sans-serif, 20px;
 
+$headers-font: "area-normal", sans-serif, 40px;
+
+
+h1 {
+  font-family: "area-normal", sans-serif;
+font-weight: 700;
+font-style: normal;
+text-align: center;
+font-size: 40px;
+
+}
+h2 {
+  font-family: "area-normal", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  text-align: left;
+  font-size: 32px;
+  text-decoration: underline;
```

Comment:
are we sure about the underline? personally i think this is a little bit outdated for a headline and prefer it without it, but i guess it is a personal preference

Reply:
I would agree with Ariane on this one but would have to see in effect to know for sure, so leave it as is for now.